debian/rules

```diff
@@ -84,6 +84,13 @@ override_dh_auto_install:
 	cp -r engine/schema/dist/systemvm-templates/* $(DESTDIR)/usr/share/$(PACKAGE)-management/templates/systemvm/
 	rm -rf $(DESTDIR)/usr/share/$(PACKAGE)-management/templates/systemvm/md5sum.txt
 
+	# Bundle in mysql_connector_python
```

Comment:
Same logic as in https://github.com/apache/cloudstack/blob/main/packaging/centos8/cloud.spec#L310